scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -462,6 +479,44 @@ static bool task_set_domain(struct task_ctx *taskc, struct task_struct *p,
 	return taskc->dom_id == new_dom_id;
 }
 
+static void set_next_dom_vtime_max(struct dom_ctx *domc, u32 dom_id, s32 cpu)
+{
+	s32 next_vtime_max_cpu = 0;
+	struct dom_cpus *domcpus;
+	u64 next_vtime_max = 0;
+	u64 *next_cpu_vtime;
+	s32 i, cpu_i;
+
+	domc->vtime_max = 0;
+
+	domcpus = MEMBER_VPTR(dom_cpus, [dom_id]);
+	if (!domcpus)
+	{
+		scx_bpf_error("Failed to lookup dom cpus[%u]", dom_id);
+		return;
+	}
+
+	bpf_for(i, 0, domcpus->nr_cpus) {
+		if (i >= MAX_CPUS)
+			continue;
```

Comment:
so this lets us avoid having to do a num cpu iteration and instead iterate # cpu in dom times instead?seems reasonable. can we add a comment explaining as much? once we have the cpumask kfunc iterator we can just use that instead and iterate over the domain's cpumask